File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// DietlyapiIntegration SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class DietlyapiIntegrationMakeContext
{
    public static function call(array $ctxmap, ?DietlyapiIntegrationContext $basectx): DietlyapiIntegrationContext
    {
        return new DietlyapiIntegrationContext($ctxmap, $basectx);
    }
}

File: rb/utility/make_context.rb
# DietlyapiIntegration SDK utility: make_context
require_relative '../core/context'
module DietlyapiIntegrationUtilities
  MakeContext = ->(ctxmap, basectx) {
    DietlyapiIntegrationContext.new(ctxmap, basectx)
  }
end
